bundles/org.openhab.binding.thekeys/src/main/java/org/openhab/binding/thekeys/internal/api/model/GatewayInfosDTO.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.thekeys.internal.api.model;
+
+/**
+ * Response DTO for / endpoint
+ * 
+ * @author Jordan Martin - Initial contribution
+ */
+public class GatewayInfosDTO {
+
+    private int version;
+    private String currentStatus;
+
+    public int getVersion() {
+        return version;
+    }
+
+    public void setVersion(int version) {
```

Comment:
Are setters really used ?
If not, record may also be a good alternative as they are final.
```
public record GatewayInfosDTO(int version, String currentStatus) {
}
```

Reply:
Good advice. I initially wrote this binding for OH3/Java 11. 
I changed to record everywhere is relevant